### PR TITLE
Various combat additions and improvements

### PR DESCRIPTION
**Pull request by A-PS1999**
I've added various new cards and some new enemies. Note that a small change to the map state is also part of this pull request, as the original text size used on the map state was affecting how text looked on the combat screen.

### CODE REVIEW

**Comment by A-PS1999** [comment]
Team, I do my best to review your pull requests promptly, so I'd appreciate it if you could reciprocate.
@JThakral20 @RicardoMiles @erer1022 @Doctor-TOTORO

File: src/engine/EndState.pde
Comment (JThakral20) [inline src/engine/EndState.pde:1]: for Line 2 - please change the variable names that start in capital case to lower case.

**Review by JThakral20 — COMMENTED** [review]
Hey looks good. I've just left a comment on the endstate for something to be changed in line 2 but other than that it looks good.

File: src/engine/EndState.pde
Comment (A-PS1999) [reply src/engine/EndState.pde:1]: Yeah, fair, I can do that. It wasn't my code originally, which is why I hadn't touched that bit. I'll make the change now.

Comment (JThakral20) [reply src/engine/EndState.pde:1]: ah right fair enough thanks for making the changes anyways. I think that's fine otherwise to merge?

**Review by JThakral20 — APPROVED** [review]
I'm not sure if my previous comment went through but fair enough thanks for making the changes anyways i think is good to merge.

**Pull request merged**